examples/test/remote/.vscode/settings.json

```diff
@@ -0,0 +1 @@
+{"files.associations":{"*.rsh":"javascript"}}
```

Comment:
You should not commit this file and should add `.vscode` to the global `.gitignore`